src/examples/wasm/server.go

```diff
@@ -3,15 +3,26 @@ package main
 import (
 	"log"
 	"net/http"
+	"os"
+	"path/filepath"
 	"strings"
 )
 
-const dir = "./html"
+var dir = "main"
 
 func main() {
-	fs := http.FileServer(http.Dir(dir))
+	if len(os.Args[1:]) == 1 {
+		dir = os.Args[1:][0]
```

Comment:
`os.Args[1]`

Reply:
I must have been so asleep when doing this. :thinking: